bot/controllers/twitchLive.js

```diff
@@ -24,7 +24,7 @@ class StreamerLiveController extends BaseController {
     const twitchURL = `https://www.twitch.tv/${twitchName}`;
     const twitchGame = message.parsed[2];
     const streamMessage = `Hey everyone! ${twitchName} is going live playing ${twitchGame}!! Check him out at ${twitchURL} !`;
-    const targetChannel = message.guild.channels.get('319548128267206666');
+    const targetChannel = message.guild.channels.get('344970351098462218');
```

Comment:
Consider making this a bit more dynamic with the following:
```
// Variable to store channel if it exists
const channel = client.channels.find('name', 'channelName');
// Check if channel exists
if (channel) {
  // Do something
}
```
This will be sure that as long as the channel name exists, it will work, as well as test to see if the channel exists.

Reply:
I agree that this would be a nice addition, and make the feature more dynamic overall.

Reply:
I also agree about changing the channel name, even if you keep with searching for a specific channel id the number you provided doesn't really give us any context on what it is. Its a 'Magic Number'. Maybe try storing it into a variable and passing that.

Reply:
I am implementing this. Thank you for the feedback.